### ISSUE TITLE
Behavior when leading "/" is ommitted?

### ISSUE THREAD

**Issue by sttawm** [gnieh/diffson#176]
What should the behavior be if the leading "/" in the `path` is omitted?

I get a pretty gnarly error in that case.

Sorry it's not a full example, I can do that if that'll be helpful. Here's what I have for now:

```
        "java.util.NoSuchElementException: JsError.get",
        "at play.api.libs.json.JsError.get(JsResult.scala:53)",
        "at play.api.libs.json.JsError.get(JsResult.scala:52)",
        "at diffson.playJson.DiffsonProtocol$.$anonfun$OperationFormat$1(DiffsonProtocol.scala:78)",
        "at play.api.libs.json.Reads$$anon$6.reads(Reads.scala:222)",
        "at play.api.libs.json.Format$$anon$6.reads(Format.scala:62)",
        "at play.api.libs.json.LowPriorityDefaultReads$$anon$8.$anonfun$reads$1(Reads.scala:268)",
```

I'm using a `JsonPatch[JsValue]` (play)

**Comment by satabin** [comment]
Hello, this is indeed not really helpful. On the top of my head, I'd say it should be a syntax error when parsing the pointer, so something is broken in the error handling here in diffson, I will fix this. Thanks for reporting.

**Comment by satabin** [comment]
After some more investigation, it appears it is a problem in the Play! JSON deserialization. The actual exception is swallowed in case of an error. I will fix this, but you will still get an exception (namely a `PointerException`) however, with a more explanatory message: "JSON pointer must start with a '/'", as per RFC-6901.

**Comment by satabin** [comment]
Behavior has been fix and I just released 4.0.3 with the fix. You should now get an exception that gives you a useful message.

**Comment by sttawm** [comment]
Wonderful! Thanks for the quick fix
